readme.md

```diff
@@ -119,14 +152,21 @@ What is the difference between the two following lines of code?
 @artist.save!
 ```
 
-> Your answer...
+> with !, the validation always runs. With !, it affects memory space.
```

Comment:
In ruby, ! does indicate mutation which I think is what you're talking about with memory. In this case however, .save will return false if the save fails and .save! will throw an error